README.md

```diff
@@ -435,8 +435,8 @@ if the HTTP calls becomes successful, they will begin to send immediately, and w
 
 * The client is optimized for high throughput. Using a single instance is no problem, but multiple instances can be created with the same key if desired.
 * Set up an `error` listener so that your app is aware of problems. Things like HTTP errors are emitted this way.
-* When shutting down your application, ensure all log entries are cleared. Services like AWS Lambda can buffer log entries,
```

Comment:
I'm fine with updating this as long as we've verified this.  I believe this came from a public user who had issues with flushing when using Lambda.  The details are long gone, though.

Reply:
the updated example code flushes properly as long as `await once(logger, 'cleared')` is called at the end of the handler.  I _suspect_ the reported behavior in the past was caused by not waiting for async operations to complete prior to the handler returning. cc: @smusali in case you have more context on that piece